tests/integration/rdb.tcl

```diff
@@ -202,6 +202,30 @@ test {client freed during loading} {
     }
 }
 
+start_server {} {
+    test {Test RDB load info} {
+        r debug populate 1000
+        r save
+        restart_server 0 true false
+        wait_done_loading r
+        assert {[s rdb_last_load_keys_expired] == 0}
+        assert {[s rdb_last_load_keys_loaded] == 1000}
+
+        r debug set-active-expire 0
+        for {set j 0} {$j < 1024} {incr j} {
+            r select [expr $j%16]
+            r setex $j 1 somevalue
+        }
+        after 1100
```

Comment:
Maybe use SET PX and a much shorter sleep?

Reply:
OK, replace with `psetex`, please approve